src/devtools/client/debugger/src/actions/project-text-search.js

```diff
@@ -77,7 +78,7 @@ export function searchSources(cx, query) {
     await dispatch(addSearchQuery(cx, query));
     dispatch(updateSearchStatus(cx, statusType.fetching));
     const validSources = getSourceList(getState()).filter(
-      source => !hasPrettySource(getState(), source.id) && !isThirdParty(source)
+      source => !ThreadFront.isMinifiedSource(source.id) && !isThirdParty(source)
```

Comment:
when prettifying happens? do you prettify upfront everything so for every minified source there is always a pretty variant available? or do u prettify on demand? Im a little bit worried that if the latter is the case then you might miss valid search results here 🤔 

could you also clarify for me, a curious reader, what a third-party source is, in this context?

Reply:
Sources are prettified by the backend and I don't know the logic for when this happens. But `ThreadFront.isMinifiedSource()` actually checks if there is a prettified variant of the given source (the name of this method is a bit misleading), so I'm sure we won't miss any search results here.
`isThirdParty()` checks if the source's URL contains `node_modules` or `bower_components`.